src/status_im/utils/ethereum/tokens.cljs

```diff
@@ -5,9 +5,11 @@
 (defn- asset-border [color]
   {:border-color color :border-width 1 :border-radius 32})
 
-(def ethereum {:name "Ethereum" :symbol :ETH :decimals 18
```

Comment:
I guess the shadowing name symbol stuff already existed so maybe this is outside of scope of PR. It should work with lexical scope regardless, so maybe it's fine in small functions. Not sure.